### ISSUE TITLE
dropout ratio for inputs

### ISSUE THREAD

**Issue by copyrightly** [tkipf/gcn#158]
In the layer.py line 170, there is `x = tf.nn.dropout(x, 1-self.dropout)`. By the documentation of tf, the argument for `tf.nn.dropout` is the ratio to drop. So should it be `x = tf.nn.dropout(x, self.dropout)`?

**Comment by shannon-np** [comment]
I believe you are correct, but at the time of writing this was not the case.

There was a change in the code for `tf.nn.dropout`; according to the [documentation](https://www.tensorflow.org/api_docs/python/tf/compat/v1/nn/dropout),
>Warning: SOME ARGUMENTS ARE DEPRECATED: (keep_prob). They will be removed in a future version. Instructions for updating: Please use rate instead of keep_prob. Rate should be set to rate = 1 - keep_prob.
